### ISSUE TITLE
tomorrow_valid not in sync

### ISSUE THREAD

**Issue by jreidel** [erlendsellie/priceanalyzer#21]
The tomorrow_valid flag will sometime show "true", even if data for tomorrow is not available from Nordpool. Everything else is working, but I use this flag to show/hide a visualization for next days prices. I haven't checked if the Nordpool integration behave in a similar way.

![image](https://user-images.githubusercontent.com/15332387/207960585-afe70856-bec8-4d36-98b0-81fc4fd2991d.png)

**Comment by erlendsellie** [comment]
Thanks for creating an issue for this bug!
This is a bug inherited from the Nordpool integration, which this integration is forked from. I've tried to fix it a few times, but don't think i've solved it quite yet. 
I'm adding some debug data to my environment so that i can figure it out.

Do you also experience the issue where raw_tomorrow  is filled with today's prices, when this happens (or otherwise)?

**Comment by jreidel** [comment]
> Do you also experience the issue where raw_tomorrow is filled with today's prices, when this happens (or otherwise)?

Yes, excacly - raw_tomorrow is allways populated with todays prices when this occur.

**Comment by erlendsellie** [comment]
I tried to fix this issue the same way the Nordpool-integration did, and just released a new version. Let me know if it worked for you or not.
I suspect it still may be an issue if there is multiple instances of this integration set up on the same Home Assistant instance. Do you just one, or multiple?
